src/pair.py

```diff
@@ -3,3 +3,25 @@
 # Part of azplugins, released under the BSD 3-Clause License.
 
 """Pair potentials."""
+
+from hoomd.azplugins import _azplugins
+from hoomd.data.parameterdicts import TypeParameterDict
+from hoomd.data.typeparam import TypeParameter
+from hoomd.md import pair
+
+
+class PerturbedLennardJones(pair.Pair):
+    """Perturbed Lennard-Jones potential."""
+
+    _ext_module = _azplugins
+    _cpp_class_name = 'PotentialPairPerturbedLennardJones'
+    _accepted_modes = ('none', 'shift', 'xplor')
+
+    def __init__(self, nlist, default_r_cut=None, default_r_on=0, mode='none'):
+        super().__init__(nlist, default_r_cut, default_r_on, mode)
+        params = TypeParameter(
+            'params',
+            'particle_types',
+            TypeParameterDict(epsilon=float, sigma=float, lam=float, len_keys=2),
```

Comment:
This is the part you need to change to match the parameters of your potential. `len_keys` stays equal to 2, it refers to the particle types.